backend/tests/Tests/DvalRepr.Tests.fs

```diff
@@ -101,7 +101,7 @@ module ToHashableRepr =
 
     testList
       "hashv2"
-      [ t [ DBytes [||] ] "diEjLGQC8oE"; t [ DBytes [| 128uy |] ] "kQHs0urT3N4" ]
+      [ t [ DBytes [||] ] "Bu4AH9NVqA0"; t [ DBytes [| 128uy |] ] "ARIiVvuJZTo" ]
```

Comment:
What could cause these hashes to change and are we supposed to change them?
The only change I could point it to is the one in Json.Vanilla.
Could you also please explain the error message `bad fsharp impl`?

Reply:
`bad fsharp impl` I think was from back when we were moving over from OCaml. This would have implied that the output was not what we expected from OCaml, and so the F# implementation was incorrect somehow.

Reply:
Yes, this change is from Json.Vanilla. Changing the format is very dangerous, which is why the test exists. Fortunately, changing this format right now is totally fine. In the future we could use a separate serializer here if we wanted to avoid this issue.

Reply:
So I think this is fine and changing the expected answer is appropriate.